portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/Form.path

```diff
@@ -518,6 +518,37 @@
 	<td></td>
 	<td></td>
 </tr>
+<!--DATA_PROVIDER_INPUT_PARAMETER-->
+<tr>
+	<td>DATA_PROVIDER_INPUT_PARAMETER_LABEL_FIELD</td>
+	<td>//div[contains(@data-field-name,'inputParameters')]//input[@type='text' and contains(@name,'inputParameters') and contains(@placeholder,'label')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>DATA_PROVIDER_INPUT_PARAMETER_NAME_FIELD</td>
+	<td>//div[contains(@data-field-name,'inputParameters')]//input[@type='text' and contains(@name,'inputParameters') and contains(@placeholder,'parameter')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>DATA_PROVIDER_INPUT_PARAMETER_TYPE_FIELD</td>
+	<td>//div[contains(@data-field-name,'inputParameters')]//div[contains(@data-field-name,'inputParameterType')]//div[contains(@class,'form-builder-select-field')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>DATA_PROVIDER_INPUT_PARAMETER_TYPE_FIELD_OPTIONS_LIST</td>
+	<td>//div[contains(@class,'dropdown-menu') and contains(@class,'show')]//li[contains(.,'${key_inputParameterType}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>DATA_PROVIDER_INPUT_PARAMETER_REPEAT_ICON</td>
```

Comment:
Good job to add this localter. But currently this locater is not called in any macro since we don't have the test to add repeat for input filed. This is more like _nice to have_ for future requirement.